Final Project App/Final Project App/ViewController.swift

```diff
@@ -9,19 +9,17 @@
 import UIKit
 import Alamofire
 
-
-
 class ViewController: UIViewController, UITableViewDataSource {
     
     //    MARK: Set variables for use in functions
     @IBOutlet var tableView: UITableView!
-    @IBOutlet var InputTextField: UITextField!
+    @IBOutlet var inputTextField: UITextField!
     
     var messages: [Message] = []
     var messagesDict: [Int:Message] = [:]
     var kbHeight: CGFloat!
-    var repeatTimer: Timer!
-    
+    var getFromServerTimer: Timer!
+    let chatServer = "http://localhost:1337/messages"
```

Comment:
It's not a "chatServer". It's the url for the GET method on the chat server. 

    let chatServerURL = "http://localhost:1337/"
    let getMessagesEndpoint = chatServerURL + "messages"